### DISCUSSION TITLE
Prefix issue template names

### DISCUSSION THREAD

**Discussion by crstauf** [upptime/upptime#335] [Ideas]
When the template is used to create a repository, the template issues carry over to the repository. I don't think there's a way around this, but prefixing the template names with something that clearly indicates they're meant for the template and not the repository would be helpful.

**Comment by AnandChowdhary** [comment]
Even better: I can remove Upptime's default issue templates as part of the update process we run every day.

**Reply by crstauf** [reply]
Taking a closer look now.

The only issue template is for bug reports... adjusting `config.yml` to make the bug report button open a new issue within the template repository, in addition to prefixing the names or changing the language of the description, may be a beautiful solution: make it easy for someone reporting an issue to access?
